From 319ff3657f05567cfdce5c899491a16696c43918 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 9 Feb 2018 18:56:54 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable CONFIG_MEMORY_STATE_TIME

f3d9c312b899 ("Implement memory_state_time, used by qcom,cpubw")

New driver memory_state_time tracks time spent in different DDR
frequency and bandwidth states.

Memory drivers such as qcom,cpubw can post updated state to the driver
after registering a callback. Processed by a workqueue

Bandwidth buckets are read in from device tree in the relevant qualcomm
section, can be defined in any quantity and spacing.

The data is exposed at /sys/kernel/memory_state_time, able to be read by
the Android framework.

Functionality is behind a config option CONFIG_MEMORY_STATE_TIME

Change-Id: I4391cc3ed42c9f332bce4a7809c6f120e2798dae
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index e13c40e26778..406dc58c712f 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -236,6 +236,7 @@ CONFIG_BLK_DEV_NVME=y
 CONFIG_ROCKCHIP_SCR=y
 CONFIG_SRAM=y
 CONFIG_UID_SYS_STATS=y
+CONFIG_MEMORY_STATE_TIME=y
 CONFIG_USB_CAM_GPIO=y
 CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
-- 
2.35.3

